build.sbt

```diff
@@ -303,15 +307,27 @@ lazy val protobuf = akkaModule("akka-protobuf")
   .disablePlugins(MimaPlugin)
 
 lazy val remote = akkaModule("akka-remote")
-  .dependsOn(actor, stream, actorTests % "test->test", testkit % "test->test", streamTestkit % "test", protobuf)
+  .dependsOn(
+    actor,
+    stream,
+    actorTests % "test->test",
+    testkit % "test->test",
+    streamTestkit % "test",
+    protobuf,
+    jackson % "test->test")
```

Comment:
surprised these are all "test->test" and not just "test". We use the jackson prod files for the tests

Reply:
I added `akka.serialization.jackson.CborSerializable` (marker interface) in src/test/scala, and as far as I know this is the way to include test sources from another project in sbt